avaframe/in3Utils/cfgUtils.py

```diff
@@ -425,20 +448,33 @@ def createConfigurationInfo(avaDir, standardCfg, writeCSV=False, specDir=''):
     return simDF
 
 
-def filterSims(avalancheDir, parametersDict, module, specDir=''):
-    """ Filter simulations using a list of parameters and a pandas dataFrame of simulation configurations """
+def filterSims(avalancheDir, parametersDict, specDir=''):
+    """ Filter simulations using a list of parameters and a pandas dataFrame of simulation configurations
 
-    standardCfg = getDefaultModuleConfig(module)
-    simDF = createConfigurationInfo(avalancheDir, standardCfg, writeCSV=False, specDir=specDir)
-    for key, value in parametersDict.items():
-        if value != []:
-            if isinstance(value, list) == False:
-                value = [value]
-            simDF = simDF[simDF[key].isin(value)]
+        Parameters
+        -----------
+        avalancheDir: str
+            path to avalanche directory
+        parametersDict: dict
+            dictionary with parameter and parameter values for filtering
+        specDir: str
+            path to a directory where simulation configuration files can be found - optional
 
-    if 'current standard' in simDF.index:
-        simDF = simDF.drop('current standard')
+        Returns
+        --------
+        simNameList: list
+            list of simNames that match filtering criteria
+    """
+
+    # load dataFrame for all configurations
+    simDF = createConfigurationInfo(avalancheDir, standardCfg='', writeCSV=False, specDir=specDir)
+
+    # filter simulations
+    if parametersDict != '':
+        for key, value in parametersDict.items():
+            simDF = simDF[simDF[key].isin(value)]
```

Comment:
not sure this is working as intended: 
If you have multiple values in parametersDict this is not working correctly, as you are actually modifying simDF. So the (random) order of parametersDict plays a role. Or am I missing something?

Reply:
This filters the simDF for all conditions in parametersDict are true, I added a comment line to explain this